test/files/neg/macro-exception.check

```diff
@@ -1,6 +1,12 @@
 Test_2.scala:2: error: exception during macro expansion:
 java.lang.Exception
 	at Macros$.impl(Macros_1.scala:6)
+#partest java20+
```

Comment:
could be `java19+` so it runs on my machine. The test was previously tweaked to avoid line numbers in stack traces. I think partest still supports `filter` pragmas that might be good enough.

Reply:
personally I can only get interested in 8, 11, 17, latest; I’ve made a conscious decision not to get finer grained than that without specific motivation

Reply:
In my "other" PR, I added a filter, which is only slightly laborious for me as an enthusiastic contributor.

I've also considered that `#partest` could be ditched in favor of splitting tests into `javaVersion: 17+` etc. Then `partest --update-check` works seamlessly again.

Reply:
hmm. not obvious to me where the other PR is, so I could see what you mean?

Reply:
I almost tagged you. This is the check file I accidentally `--updated` and overwrote pragmas. https://github.com/scala/scala/pull/10378/commits/3c69c6113c1e308998159836cc8179014be7dd74

Reply:
a karma ran over my pragma